test/typecheck/README

```diff
@@ -19,5 +19,5 @@ Things go/types errors on that go build doesn't:
     These should be ignored and reported upstream if applicable.
     - type checking mismatches between staging and generated types
 Things go build fails on that we don't:
-  Please send examples of this to rmmh@ and extend this list.
+  Please send examples of this to @rmmh and extend this list.
```

Comment:
Please delete this line completely. rmmh isn't heavily involved in the project anymore.

Reply:
I've just deleted the line.